.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
```

Comment:
Did you add this file? I don't think this should be added

Reply:
It wasn't added intentionally. Will find a way to remove.